src/background.js

```diff
@@ -157,24 +146,12 @@
   });
 
   // Check whether new version is installed
-  chrome.runtime.onInstalled.addListener(details => {
-    switch (details.reason) {
-      case 'install':
-        chrome.runtime.openOptionsPage();
-        break;
-      default:
-        updateBadge();
-    }
+	chrome.runtime.onInstalled.addListener(details => {
+		if (details.reason === 'install') {
+			chrome.runtime.openOptionsPage();
```

Comment:
Well it didn't before (`default` case is not raised during installation), and it shouldn't: settings are not currently reviewed by the user, and an alarm is already created.